### PR TITLE
BLIND_EASY flag for most quern recipes

### PR DESCRIPTION
**Pull request by Brambor**
#### Summary
```SUMMARY: Balance "BLIND_EASY flag for quern recipes"```

#### Purpose of change
Some recipes using `quern` or `mortar_pestle` make sense to be possible to be done blind.

#### Describe the solution
Recipes adjusted (both `quern` and `mortar_pestle` versions):
- cornmeal
- flour
- meal_bone
- meal_bone_tainted
- meal_chitin_piece

Added `BLIND_EASY` flag to said recipes.

#### Describe alternatives you've considered
Forcing the player to have a light source while crafting said recipes, which seems redundant.

#### Additional context
I might have missed something, oatmeals requires `surface_heat` so I figured you cannot do it blindly.

There is `mods\Tolerate_This\recipe_food.json` where the recipes are repeated (quern versions), I do not quite understand the purpose and the code in general as I didn't inspect it. I changed it there as well. Am I doing it right?

### CODE REVIEW

**Comment by Brambor** [comment]
Yeah, I screw up with that 'fixed typo in door curtains' commit.

**Comment by Qrox** [comment]
You should probably rebase this PR, otherwise when this is merged the changes in commit b8bcca8 (the one you proposed in #25160) will be lost. (When the revert commit is merged, it will undo the changes it reverts)

File: data/json/recipes/recipe_companion.json [hunk 1]
```diff
@@ -224,6 +224,7 @@
     "time": 56250,
     "autolearn": false,
     "never_learn": true,
+    "flags": ["BLIND_EASY"],
```
Comment (ZhilkinSerg) [inline data/json/recipes/recipe_companion.json]: All files you have touched should be linted (you can use http://dev.narc.ro/cataclysm/format.html for that) - 
Should be `[ "BLIND_EASY" ]` (i.e. single whitspace after `[` and before `]` should be added).

Comment (Brambor) [reply data/json/recipes/recipe_companion.json]: Crap, I supposed that where I copied it from it would be ok. Ok.
How is everything not linted yet?

**Comment by Brambor** [comment]
@Qrox Looks much more confusing now, but is it ok?

**Comment by cainiaowu** [comment]
Well, check the "change files", it is apparently not. Might as well create a new PR.

**Comment by Brambor** [comment]
Yeah, ok...

**Pull request closed without merging**